common/os.cpp

```diff
@@ -39,7 +39,7 @@
 #endif
 
 #include <GLFW/glfw3.h>
-#include "win/win-helpers.h"
+#include "utilities/os/h_result.h"
```

Comment:
It's `HRESULT` without a `_` -- can you make it `hresult.h` please?

Reply:
done